docs/installation/pipx-based.rst

```diff
@@ -140,17 +178,6 @@ To install ``pyenv``, follow these steps:
 
         Finally open a normal PowerShell and re-run the above installation command.
 
-After completing the above steps, you will need to close the terminal and re-open it again. After that, to check if things work run:
-
-.. code-block:: bash
-
-    pyenv --version
-
-You should get something similar to:
-
-.. code-block:: output
-
-    pyenv 3.1.1
 
 Actually installing Python
```

Comment:
I know you didn't write this line, but I think this sounds *too* informal!


How about:

```suggestion
Installing your chosen Python version
```